public/index.html

```diff
@@ -45,108 +45,6 @@
         <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
 
-    <!-- Register Payment App SW -->
-    <script>
-    /**
-     * Utilities for registering/unregistering the Payment App service worker.
-     */
-
-    const SERVICE_WORKER_URL = '/pay/sw-bobbucks.js'
-
-    // Adds the BobBucks default instrument.
-    function addInstruments(registration) {
-      registration.paymentManager.userHint = "test@bobbucks.dev";
-      return Promise.all([
-        registration.paymentManager.instruments.set(
-            "dc2de27a-ca5e-4fbd-883e-b6ded6c69d4f",
-            {
-              name: "My Bob Pay Account",
-              icons: [{
-                src:"/pay/bobbucks.png",
-                sizes:"32x32",
-                type:"image/png"}
-              ],
-              method: "https://bobbucks.dev/pay"
-            }),
-        registration.paymentManager.instruments.set(
-            "new-card",
-            {
-                name: "Add a new card to BobBucks",
-                method: "basic-card",
-                capabilities: {
-                    supportedNetworks: ["visa", "mastercard", "amex", "discover"],
-                    supportedTypes: ["credit", "debit", "prepaid"]
-                }
-            }),
-      ]);
-    };
-
-    // Shows/hides the webpage controls for installing/removing the service
-    // worker.
-    function showBobBucksStatus(enabled) {
-        showBobBucksError("");
-        var buttonText = enabled ?
-                'Web App Installed &#9989;' : 'Install Web App';
-        document.getElementById("installbutton").innerHTML = buttonText;
-
-        document.getElementById("installed").classList.toggle(
-                'invisible', !enabled);
-        document.getElementById("notinstalled").classList.toggle(
-                'invisible', enabled);
-    }
-
-    // Shows/clears an error related to the service worker installation.
-    function showBobBucksError(errorMessage) {
-        document.getElementById('bobbuckserror').innerHTML = errorMessage;
-    }
-
-    // Registers the payment app service worker by installing the default
-    // instruments.
-    function registerPaymentAppServiceWorker() {
```

Comment:
Please remove this function's usage from line 115. 

It would be good to change line 115 from:

```
<a href="#installit" onclick="registerPaymentAppServiceWorker()">
                    Install BobBucks Web Payment App</a>
```

To:

```
Web Payment Apps are installed just-in-time (JIT) during the first payment request.
Install BobBucks by navigating to a
<a href="https://rsolomakhin.github.io/pr/bob/">Sample Merchant for BobBucks</a>
and initiating a demo payment request. No actual payment will be processed on the demo page.
```